policy-service/src/main/java/uk/gov/gchq/palisade/service/policy/web/PolicyController.java

```diff
@@ -139,7 +139,7 @@ public void setTypePolicyAsync(@RequestBody final SetTypePolicyRequest request)
     public void initPostConstruct() {
         // Add example Policies to the policy-service cache
         policyConfig.getPolicies().stream()
-                .map(cacheWarmer -> cacheWarmer.policyWarm(userConfig.getUsers()))
+                .map(cacheWarmer -> cacheWarmer.build(userConfig.getUsers()))
```

Comment:
```suggestion
                .map(prePopulation -> prePopulation.build(userConfig.getUsers()))
```
Might be worth updating this to reflect the updated class name.